docs/source/guides/contributor/chapters/runners.rst

```diff
@@ -568,3 +568,96 @@ for runnables with kind ``foo``.
 .. literalinclude:: ../../../../../examples/nrunner/runners/avocado-runner-foo
    :language: python
    :linenos:
+
+
+Runners messages
+----------------
+
+Every runner have to send information about the run to the avocado
+core. Those informations are sent by messages which have different
+types based on the information which they are transmitting. Avocado
+understands three main types of messages, the start, running, and
```

Comment:
I'd use bullet points here for the three message types, just to make the visualization very clear.  Something like:

Avocado understands three main types of messages:
* `start` (required)
* `running`
* `finished` (required)